File: content/learn/arrays-easy/03-is-sorted/starter.cpp
#include <iostream>
#include <vector>
using namespace std;

bool isSorted(const vector<int>& a) {
    // TODO: return true iff a is non-decreasing.
    return false;
}

int main() {
    int n;
    cin >> n;
    vector<int> a(n);
    for (int i = 0; i < n; i++) cin >> a[i];
    cout << (isSorted(a) ? "true" : "false") << "\n";
    return 0;
}

File: content/learn/arrays-easy/03-is-sorted/starter.js
function isSorted(a) {
  // TODO: return true iff a is non-decreasing.
  return false;
}

const n = parseInt(readline(), 10);
const a = n > 0 ? readline().split(" ").map(Number) : [];
print(isSorted(a) ? "true" : "false");
